### ISSUE TITLE
PvP Rework - CSAT (Arid)

### ISSUE THREAD

**Issue by ComboTombo** [ComboTombo/A3-Antistasi#6]
Primary Weapon will be set to the Katiba Carbine with an ACO scope and an IR laser.

_CSAT special forces would use a lightweight, mobile, short-barreled version of the Katiba_

Secondary weapon will be set to the Rook.

_It's the vanilla equivalent of the MP-443_

Uniforms will be set to the Recon Fatigues (Hex). 

Vests will be set to the LBV Harness.

Each vest will have 2 smoke nades, 1-2 fragmentation nades, 4-5 mags, and a pistol mag.

Helmets will be set to the Assassin helmet.

_It's a spec ops helmet that provides adequate protection. Not too heavy & strong, and not too light & weak._ 

Team Leader gets a GPS and a rangefinder.

_The reason why the team leader gets a rangefinder is that it incentivizes teamwork between the team leader and marksman, or team leader and AT._

NVGs will be set to the black vanilla NVGs.

### SPECIAL FOR AT AND AAT ###

AT gets an RPG-32 with an RPG-32 rocket loaded and another rocket in a backpack.

AAT gets two RPG-32 rockets in their backpacks.

_Basically an ammo bearer. If vanilla had disposable AT sticks, he would've had one._

### SPECIAL FOR MACHINEGUNNER ###

MG gets a Zafir with a holosight and an IR laser pointer

_It's the only OPFOR machinegun that doesn't require the Marksman DLC._

MG vest gets one 150rnd box

_The already relatively heavy machinegunner will only get 300 rounds of ammunition_

### SPECIAL FOR MARKSMAN ###

Marksman gets a Rahim with a DMS scope

_Only CSAT DMR that doesn't require the Marksman DLC_

Marksman vest has 9 10rnd magazines.

_All marksmen loadouts will get exactly 100 rounds of ammunition. If you can't hit a target with 100 rounds, a high powered scope, and a marksman rifle, you should give the role to someone else that can_
